test/simulation/simulation/logs.py

```diff
@@ -1,13 +1,83 @@
 # LOG_ADD_PEER = 'rpcserver:AddPeer: ip=192.168.11.69, port=10001, seed=1'
-LOG_ADD_PEER_FMT = 'rpcserver:AddPeer: ip={}, port={}, seed={}'
-LOG_ADD_PEER_FINISHED = 'rpcserver:AddPeer: finished'
+from enum import (
+    Enum,
+    auto,
+)
+
+
+class RPCLogs(Enum):
+    LOG_ADD_PEER_FMT = auto()
+    LOG_ADD_PEER_FINISHED = auto()
+    LOG_BROADCAST_COLLATION_FMT = auto()
+    LOG_BROADCAST_COLLATION_FINISHED = auto()
+    LOG_SUBSCRIBE_SHARD_FMT = auto()
+    LOG_SUBSCRIBE_SHARD_FINISHED = auto()
+    LOG_UNSUBSCRIBE_SHARD_FMT = auto()
+    LOG_UNSUBSCRIBE_SHARD_FINISHED = auto()
+    LOG_DISCOVER_SHARD_FMT = auto()
+    LOG_DISCOVER_SHARD_FINISHED = auto()
+    LOG_REMOVE_PEER_FMT = auto()
+    LOG_REMOVE_PEER_FINISHED = auto()
+    LOG_STOP_FMT = auto()
+    LOG_STOP_FINISHED = auto()
+    LOG_BOOTSTRAP_FMT = auto()
+    LOG_BOOTSTRAP_FINISHED = auto()
+
+
+class OperationLogs(Enum):
+    LOG_RECEIVE_MSG = auto()
+
+
+# The regex for the list of elements: empty list, or 1 or more words, delimited by whitespaces
+# E.g. [], [1], [1 2], [shardCollations_2]
+REGEX_LIST = r'\[(?:|\w+(?: \w+)*)\]'
```

Comment:
`\w` in `[]` and `(?:)` might not work in `grep`. Probably need to modify them

Reply:
Sorry I'm wrong. [Non-capturing groups](https://www.regular-expressions.info/brackets.html) `(?:)` are supported in `grep`.